modules/ROOT/pages/restarting-a-node.adoc

```diff
@@ -1,78 +1,118 @@
 
-[NOTE]
-To preserve your data in case of system failure, ensure that your system is backed up before attempting node maintenance.
-[NOTE]
-The steps below must be performed on a single Anypoint Platform PCE node at a time.  If maintenance is required on more than one node, the entire process must be completed on a single node before the next node is addressed.
+== Cluster and Application Updates
 
-== Drain the Node
+[IMPORTANT]
+The following steps must be performed on one Anypoint Platform Private Cloud Edition node at a time to minimize or eliminate component breakage during the update process. The entire process must be completed on each node before the next node is addressed. Verify the health status of all cluster components and applications after each node update before proceeding with the next node.
 
-Before any maintenance or restart of a node can be performed, it must be removed from the cluster and its workload reassigned to the other nodes in the cluster.
+=== Verify Cluster Health Status
 
-. Check if the firewalld service is running and enabled:
+Verify that the cluster is healthy before starting the process. Otherwise, many of the assumptions made in this process are not valid.
+
+. Verify the output from the following commands:
++
+```
+gravity status
+etcdctl cluster-health
+```
++
+If the output from the previous commands mentions a `degraded` state or `errors`, fix all issues before proceeding.
+
+. Verify that all pods are running:
 +
-----
-service firewalld status
-----
-. If firewalld is running, stop, disable, and mask it:
+```
+gravity get pod -o wide--all-namespaces | grep -vi Running
+```
 +
-----
-sudo systemctl stop firewalld
-sudo systemctl disable firewalld
-sudo systemctl mask firewalld
-----
-. Retrieve the *name* of target node:
+If the output from the previous command lists any pod not in "Running" state, then fix all issues before proceeding.
+
+=== Install Updates
+
+After verifying that the cluster is healthy, select a node on which to perform the following steps:
+
+. Retrieve the *name* of the target node:
 +
-----
+```
 kubectl get nodes
-----
+```
 . Drain the node:
 +
-----
+```
 kubectl drain --delete-local-data --ignore-daemonsets <node_name>
-----
-. Stop Gravity
+```
+. After the node is drained, identify the gravity unit name:
 +
-----
-systemctl list-units | grep "planet-master"
-  # note the full name of the Gravity "planet-master" unit
-systemctl stop <garvity_unit_name>
-----
+```
+sudo systemctl list-units --plain | awk '/planet-master/{print $1}'
+```
+The previous command outputs a `systemd` unit name, which is used as the `$unit_name` value in the next step.
+. Stop the gravity unit:
++
+```
+systemctl stop $unit_name
+systemctl status $unit_name
+```
+. After the `status` subcommand indicates the gravity unit is stopped, install the needed updates and reboot the server.
 
-== Apply Node Maintenance
+=== Perform Post-update Check
```

Comment:
Verify the Installation

Reply:
Try to add some snippet of text here that provides the context and leads to the first step of this third task.